chicken-sloths/bangazon-site-sprint3
# Kb products by type

# Description
Displays products by type according to the ID in the route.

## Related Ticket(s)
fixes #4, fixes #38

## Expected Behavior
If you go to `/categories/:id`, it should show you all the non-sold out, non-deleted products in that category.

## Steps to Test Solution
1. `npm run db:gen`
1. `npm start`
1. Go to `/categories/3`
1. You should see two items with 21 and 28 items left respectively.
1. There should be links to the individual product pages, whose routes are not hooked up.
